app/frontend/src/features/donations/DonationPage.tsx

```diff
@@ -88,6 +88,7 @@ export default function DonationPage() {
           id="recurring"
           control={control}
           name="recurring"
+          rules={{ required: "This field is required." }}
```

Comment:
Currently the value of the field is a boolean, true if recurring. So selecting one-time fails with "This field is required".
You can change it so the onChange line just does `onChange(e.target.value)` and adjust how `onSubmit` works accordingly, or have a custom validation rule that differentiates between undefined and false.